recipe/meta.yaml

```diff
@@ -26,30 +26,17 @@ requirements:
   build:
   - cmake>=3.10
   - eigen
-#  - eigen=3.3.3
   - gsl
-#  - gsl==2.2.1
-  - numpy
-  - openblas
-#  - openblas==0.2.19
-  - python
-#  - python==3.6
-  - python-dateutil
-  - pvl
-  - spiceypy
   run:
-  - cmake>=3.10
   - eigen
-#  - eigen=3.3.3
   - gsl
-#  - gsl==2.2.1
+  - netowrkx
   - numpy
   - openblas
-#  - openblas==0.2.19
   - python
-#  - python==3.6
   - python-dateutil
   - pvl
+  - pysis
```

Comment:
In the short term I think merging this is fine. I am going to open an issue about the pysis dependency. From what I am seeing in util.py, pysis is not actually required. In fact, an ISIS install is not even required. What is needed is an ISIS data area and some parsing that can be done. 

My thinking here is that I really would like to avoid a circular ISIS<->ALE dependency.